### PR TITLE
Nancy Ng- Description Changes

### PR DESCRIPTION
**Pull request by njng2**

### CODE REVIEW

File: f21/ps.html [hunk 1]
```diff
@@ -565,10 +565,19 @@ <h2><u> Lecture / Lab 5</u></h2>
...
 <a href="https://www.datacamp.com/community/tutorials/python-numpy-tutorial">Datacamp Numpy Tutorial</a>,  <a href="lab3.html">Lab 3</a>,  <a href="lab4.html">Lab 4</a> & <a href="../f20.html">Lecture 4</a></b> <br>
 <p><b>REVIEW from Lecture/Lab 4</b>
-<p>Write a program that creates an image of purple and cyan horizontal stripes. Think color not shade: Purple = red and blue only, no green. Cyan = green and blue only, no red.
+<p>Write a program that creates an image of khaki and rosy horizontal and vertical stripes. 
+  <br>
+  <br>The color codes are below. 
+  <br>
+  rgb(188,143,143), #BC8F8F - Rosy Brown 
+  <br>
+  rgb(240, 230,140) #f0e68c - khaki
```
Comment (tligorio) [inline f21/ps.html]: For consistency, please keep all the hex codes in capital letters. Thanks!

File: f21/ps.html [hunk 2]
```diff
@@ -565,10 +565,19 @@ <h2><u> Lecture / Lab 5</u></h2>
...
+  <br>
+  rgb(188,143,143), #BC8F8F - Rosy Brown 
+  <br>
+  rgb(240, 230,140) #f0e68c - khaki 
+  <br>
+  Use the color converter to get the RGB% from hex:https://andylangton.co.uk/tools/colour-converter
+  <br>
   <br>Your program should ask the user for the size of your image, the name of the output file, and create a <tt>.png</tt> file of stripes.  For example, if the user enters 10, your program should create a 10x10 image, alternating between purple and cyan stripes.
 
 <p>
```
Comment (tligorio) [inline f21/ps.html:583]: The sample output here only asks for the output file, but the description above says the program should as for both image size and output file, can you please make it consistent?

File: f21/ps.html [hunk 3]
```diff
@@ -1066,57 +1075,54 @@ <h2><u> Lecture / Lab 8</u></h2>
...
-
-The three most dinosaur-populated countries were:
-North America    274
-China            217
-South America    149
-Name: Country, dtype: int64
+Enter file name: vgsales.csv
+There are 16598 total games
+
+The number of game genres are
```
Comment (tligorio) [inline f21/ps.html]: Please make this "The number of games in each genre are:"
Please update the autograder  accordingly too if necessary. Thank you!

File: f21/ps.html [hunk 4]
```diff
@@ -1700,31 +1724,25 @@ <h2><u> Lecture / Lab 13</u></h2>
...
+  How much are you spending in a week for coffee: 40.50
+  Budget left after week 1	359.5
+  Budget left after week 2	319
+  Budget left after week 3	278.5
+  Budget left after week 4	238
 </pre>
 
-<p> Note: the autograder is expecting each line to begin with "Year " and the corresponding number for the year.
-<p><b>IMPORTANT:</b> For grading purposes, set the precision to two decimal places. To do so, add <tt>#include &lt;iomanip&gt;</tt> at the top of the file, and <tt>cout << fixed << setprecision(2); </tt>.
+<p> Note: the autograder is expecting each line to begin with "Week" and the corresponding number for the week
```
Comment (tligorio) [inline f21/ps.html]: I'm not sure what this means that the autograder expects each line to begin with "Week", in the sample output each line begins with "Budget left..." what you expect the students to do should be consistent with the sample output, which is it? Or am I not understanding the note?

**Review by tligorio — CHANGES_REQUESTED** [review]

File: f21/ps.html (hunk 4 above)
Comment (njng2) [reply f21/ps.html]: Sorry about that. You don't need the line to start with "week" or "budget left after week...".

Comment (njng2) [reply f21/ps.html]: i removed that line. i tested it without that line on the autograder.

**Pull request merged**